lib/main.dart

```diff
@@ -235,6 +246,9 @@ class App extends StatelessWidget {
             '/settings/audio-sources': (context) => const AudioSourcesScreen(),
             '/settings/quick-links': (context) => const QuickLinksScreen(),
             '/settings/backup': (context) => const BackupScreen(),
+            '/settings/events-config': (context) => const EventConfigScreen(),
```

Comment:
`config` isn't necessary as it's under `/settings`. `/settings/events` is sufficient.